Form-Controls/index.html

```diff
@@ -1,28 +1,96 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <title>My form exercise</title>
-  <meta name="description" content="">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="styles.css">
-</head>
-<body>
-  <header>
-    <h1>Product Pick</h1>
-  </header>
-  <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>My form exercise</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header>
+      <h1>Product Pick</h1>
+    </header>
+    <main>
+      <form>
+        <fieldset>
+          <label for="first-name" class="label">First Name</label>
+          <input
+            type="text"
+            name="first-name"
+            required
+            minlength="2"
+            class="input"
+          />
+        </fieldset>
 
-  </main>
-  <footer>
-    <!-- change to your name-->
-    <h2>By HOMEWORK SOLUTION</h2>
-  </footer>
-</body>
+        <fieldset>
+          <label for="last-name" class="label">Last Name</label>
+          <input
+            type="text"
+            name="last-name"
+            required
+            minlength="2"
+            class="input"
+          />
+        </fieldset>
 
-</html>
\ No newline at end of file
+        <fieldset>
+          <label for="email" class="label">Email Address</label>
+          <input type="text" name="user-email" required class="input" />
+        </fieldset>
+
+        <fieldset>
+          <p>Color</p>
```

Comment:
This should be a `legend` element instead of `p`. 
A screen reader will then tell the user that the inputs in the fieldset are described by the text in the `legend`.
Have a look at [this example](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio#defining_a_radio_group)